Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2.yml

```
@@ -105,6 +105,15 @@ args:
   name: page
   required: false
   secret: false
+- default: false
+  description: |-
+    The number of events to return from the alert JSON. The default is 0, which returns all events.
+    Note that the trimming is from the top of the list, regardless of event time or other properties.
```

`top of the list` is ambiguous.
```suggestion
    Note that the trimming is from the end of the list, regardless of event time or other properties.
```